department-of-veterans-affairs/caseflow
# Daily docket time updates

Connects #7620 

### Description
This PR adds frontend functionality for hearing time updates on the daily docket. A future PR will save the information to the backend.

### Testing Plan
1. Go to hearings/schedule and click through to a daily docket
1. Update a hearing's time and confirm the new state is saved in Redux
